week-2/Homework/mandatory/2-function-creation.js

```diff
@@ -38,7 +58,19 @@ Write a function that:
 */
 
 function formatPercentage(arr) {
-  
+  let myArray = arr;
```

Comment:
Note that it might not be necessary to do this in this case. You can end up mixing `myArray` and `arr`, which did indeed happen. It might be cleaner to remove this line and avoid the potential confusion.

Reply:
ok